pandas/plotting/_core.py

```diff
@@ -2704,18 +2704,56 @@ def __call__(self, x=None, y=None, kind='line', ax=None,
 
     def line(self, x=None, y=None, **kwds):
         """
-        Line plot
+        Plot `DataFrame` columns as lines.
+
+        This function is useful to plot lines using `DataFrame`'s values
```

Comment:
I think sphinx complains if you do you have something right after a \`. Just do DataFrame without the backticks.